testing/gmock/APKBUILD

```diff
@@ -0,0 +1,51 @@
+# Contributor: Daniel YC Lin <dlin.tw@gmail.com>
+# Maintainer: Daniel YC Lin <dlin.tw@gmail.com>
+pkgname=gmock
+pkgver=1.8.0
+pkgrel=0
+pkgdesc="Google Mock - A library for writing and using C++ mock classes"
+url="https://github.com/google/googletest"
+arch="all"
+license="BSD"
+depends="libgcc bash python2 gtest"
```

Comment:
Dependency on libgcc is auto-discovered via dynamic links (`so:libgcc_s.so.1`), so it’s not needed (should not be) to specify it here.